lib/uwb/include/uwb/UwbSession.hxx

```diff
@@ -143,6 +143,24 @@ public:
     std::vector<::uwb::protocol::fira::UwbApplicationConfigurationParameter>
     GetApplicationConfigurationParameters();
 
+    /**
+     * @brief Get the Application Configuration Parameters object
+     *
+     * @param requestedTypes
+     * @return std::vector<::uwb::protocol::fira::UwbApplicationConfigurationParameter>
+     */
+    std::vector<::uwb::protocol::fira::UwbApplicationConfigurationParameter>
+    GetApplicationConfigurationParameters(std::vector<::uwb::protocol::fira::UwbApplicationConfigurationParameterType> requestedTypes);
+
+    /**
+     * @brief Set the Application Configuration Parameters object
+     *
+     * @param params
+     * @return void
```

Comment:
I don't think the `@return void` line is necessary here, nor in the other locations

Reply:
This one is still here